lib/Transforms/ConvertIfToSelect/BUILD

```
@@ -0,0 +1,43 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library")
+
+package(
+    default_applicable_licenses = ["@heir//:license"],
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "ConvertIfToSelect",
+    srcs = ["ConvertIfToSelect.cpp"],
+    hdrs = ["ConvertIfToSelect.h"],
+    deps = [
+        ":pass_inc_gen",
+        "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:SCFDialect",
+        "@llvm-project//mlir:Support",
+        "@llvm-project//mlir:Transforms",
+    ],
```

It probably gets the Arith dependency transivitely from SCF or something else already, but I think the pass pass should explicitly depend on ArithDialect

```suggestion
    deps = [
        ":pass_inc_gen",
        "@llvm-project//mlir:IR",
        "@llvm-project//mlir:Pass",
        "@llvm-project//mlir:ArithDialect",
        "@llvm-project//mlir:SCFDialect",
        "@llvm-project//mlir:Support",
        "@llvm-project//mlir:Transforms",
    ],
```